spikely/pipeman/pipeman.py

```diff
@@ -50,7 +52,11 @@ def append(self, text):
         self.output.append(text)
 
     def stdout_ready(self):
-        text = bytearray(self.process.readAllStandardOutput()).decode()
+        if sys.platform.startswith('win'):
+            text = bytearray(self.process.readAllStandardOutput()).decode(os_encoding)
+        else:
+            text = bytearray(self.process.readAllStandardOutput()).decode()
+
```

Comment:
Does it make sense to use an os specific os_encoding parameter for both Linux and Windows?  decode() without a parameter defaults to utf-8 I believe.  Maybe we should do the same thing you are doing for Windows on Linux as well, and remove the if/else logic.

Reply:
Yes, it makes sense. Actually, that's what I did in the first fix-up. But, as I haven't tested it on Linux, I left this part to a safe zone.

IMO, all if/else could be deleted if the lines for Win could run on Linux.